### ISSUE TITLE
[Feature Request]: Support for Ross Vision Tallies

### ISSUE THREAD

**Issue by philipdroberts** [josephdadams/TallyArbiter#580]
### What feature would you like to see added to TallyArbiter?

Have you considered adding support for Ross Vision Tallies? 

The Vision uses a protocol derived from the Grass Valley Kalypso protocol over RS422.
https://wwwapps.grassvalley.com/docs/Manuals/switchers/kalypso/071-8063-08.pdf

The differences are limited the input and output source numbering and I do have that documented also.

Thanks

Philip

**Comment by josephdadams** [comment]
Is it only over serial?

**Comment by philipdroberts** [comment]
Yes, only serial. Its RS422 but I'm currently using a RS422 to RS232
converter and running it into a PLC
